framework/auth/views.py

```diff
@@ -87,6 +103,8 @@ def reset_password_post(uid=None, token=None):
 
     # Check if request bears a valid pair of `uid` and `token`
```

Comment:
This comment no longer makes sense here, maybe move it down above the real check